README.md

```diff
@@ -16,3 +16,16 @@ and you will find a description of the extensions beyond the spec here
 
 Our initial goal is to demonstrate the work on a k8s cluster, but later plan to
 support the generic use case.
+
+## etcd schema used in the project
+In the project we use the following etcd key formation convention:
+* all key parts are slash ("/") separated
+* keys start from deployment defined prefix, which can contain several slash separated sub-parts.
+* the next key part defines the deployment service, e.g. "sbdb" or "nbdb"
+* after that, we have a database server name, e.g. `OVN_Northbound`, `OVN_Southbound`, `_Server`. There  is an 
+  additional internal entry `_` for locks and comments.
+* next key element defines table names according to the relevant schema. Locks and comments are stored in the internal 
+  database in `_locks` and `_comments` entries.
+* the last key element is the `uuid` of the table row. This element can be the lock or comment id as well.
+* in order to guarantee that `_Server/Database` will contains only entries per running databases, its last key element
+  is not uuid, but the database server name.
```

Comment:
I would rather use semantics as follows to define (and below you can elaborate on each part):
- data:           <prefix>/<service>/<dbname>/<table>/<uuid> --> <row>
- locks:          <prefix>/<service>/_/_locks/<lockid> --> ?
- comments: <prefix>/<service>/_/_comments/<timestamp> --> <comment>